mne/realtime/fieldtrip_client.py

```diff
@@ -1,353 +1,351 @@
-# Author: Mainak Jas
-#
-# License: BSD (3-clause)

```

Comment:
This file was auto-changed by Spyder because it had mixed line endings. I can revert if necessary.